### DISCUSSION TITLE
LoadBalancer Service Type

### DISCUSSION THREAD

**Discussion by TJPras** [arkmq-org/arkmq-org-broker-operator#608] [Q&A] [answered]
Hi, I would like to be able to create a LoadBalancer service instead of a ClusterIP service so that I can access the broker Management Console from outside of the cluster. Is this possible, and if so how would I do that?

I have read the discussion [here](https://github.com/artemiscloud/activemq-artemis-operator/discussions/554#discussioncomment-5466674) and it's not clear to me how I can accomplish this. I also see that the issue created from this discussion is still open [#557](https://github.com/artemiscloud/activemq-artemis-operator/issues/557).

Thank you,
TJ

**Comment by brusdev** [comment, marked as answer]
@TJPras the feature will be added by #557 in the meantime you could manually create a service using the same selector used by the operator, i.e for an ActiveMQArtemis CR with the name `ex-aao` you could use the following selector:
```
  selector:
    ActiveMQArtemis: artemis-mbp
    application: artemis-mbp-app
```
